### ISSUE TITLE
[User story proposal] Xarray for energy analytics

### ISSUE THREAD

**Issue by dcherian** [xarray-contrib/xarray.dev#358]
This is a proposal for a energy analytics use case story for Xarray. The story will be published on Xarray's blog (https://xarray.dev/blog). 

### Why?

Xarray is used in a number of energy applications today (e.g. MDSIO). A user story blog post showing how Xarray can be used to represent and analyze oil & gas / wind energy data would help socialize the concept across the two communities.

### What?

We are targeting a short and non-technical post that illustrates how Xarray is (or could be) used in your energy analysis context. Below is a template outline that you should feel free to modify:

1. Who am I?
2. What problem am I trying to solve?
3. How does Xarray help?
4. Why did I choose Xarray? 
5. Current pain points (or ways that Xarray could better serve your use case)
6. Technology I use around Xarray
7. Anything else to know?
8. Links and references

Feel free to insert images or short code blocks if they help you tell your story.

### How?

Xarray's blog using Markdown with some front-matter. You can copy one of the [existing posts](https://github.com/xarray-contrib/xarray.dev/tree/main/src/posts) to get started or you can write in a google-doc-like-thing and ask us to do the markdown formatting. If you copy a previous post, use a name like `user-story-brain-imaging` or whatever makes sense for your application.

xref: https://github.com/xarray-contrib/xarray.dev/issues/272

**Comment by dcherian** [comment]
@srib and @tasansal would either of you be up for contributing a short post around https://github.com/TGSAI/mdio-python or related projects.

**Comment by tasansal** [comment]
Yes! We have a lot of material for wind resource assessment but have to check with our employer first, about how much we can share. 

Most of our wind ETL pipeline is Xarray Dask Zarr and AirFlow

**Comment by srib** [comment]
Yes! We should be able to contribute something to the blog both in seismic data management and wind resource assessment as @tasansal mentioned.

There is a plan to rewrite mdio using xarray backend and simplify mdio considerably. There is an open issue and some discussion on [mdio-python#33](https://github.com/TGSAI/mdio-python/issues/33).

**Comment by jhamman** [comment]
I'm fine combining these two ideas into one if it is easy to do. Another option here would be to split this into two shorter posts. One focusing on wind resource assessment that makes use of xarray. And a second demoing the mdio backend.
